src/App.tsx

```diff
@@ -1,7 +1,25 @@
 import React from "react";
+import Navbar from "./components/Navbar/navbar";
+import About from "./components/pages/about";
+import Home from "./components/pages/home";
+
 
 function App() {
-  return <div className="App">To be implemented</div>;
+  let component;
+  switch (window.location.pathname) {
```

Comment:
Let's try something different with navigation. I would like you to explore `react-router-dom` and the React router. This is a modern, secure way to navigate between pages. Some resources:
- [My old website that used routing - code base](https://github.com/MDixey17/personal-website/blob/main/src/App.js)
- [W3Schools](https://www.w3schools.com/react/react_router.asp)

Reply:
I didn't know whether or not to use react-router-dom for external sites like twitch, twitter and discord. I left them as anchor tags for now.

Reply:
External links like Twitter, Twitch, Discord, etc. should always be anchor tags, so you are good there